### PR TITLE
feat: standardized timestamp comparison

### PR DESCRIPTION
**Pull request by 0xShaito**
# 🤖 Linear

Closes OPO-668
Closes OPO-664

### CODE REVIEW

File: solidity/contracts/modules/resolution/PrivateERC20ResolutionModule.sol [hunk 1]
```diff
@@ -85,8 +85,8 @@ contract PrivateERC20ResolutionModule is Module, IPrivateERC20ResolutionModule {
       _escalation.startTime + _params.committingTimeWindow,
       _escalation.startTime + _params.committingTimeWindow + _params.revealingTimeWindow
     );
-    if (block.timestamp <= _revealStartTime) revert PrivateERC20ResolutionModule_OnGoingCommittingPhase();
-    if (block.timestamp > _revealEndTime) revert PrivateERC20ResolutionModule_RevealingPhaseOver();
+    if (block.timestamp < _revealStartTime) revert PrivateERC20ResolutionModule_OnGoingCommittingPhase();
+    if (block.timestamp >= _revealEndTime) revert PrivateERC20ResolutionModule_RevealingPhaseOver();
```
Comment (xorsal) [inline solidity/contracts/modules/resolution/PrivateERC20ResolutionModule.sol:89]: What do you think about forcing the use of `<` and inverting this comparison?

```
if (_revealEndTime < block.timestamp) revert PrivateERC20ResolutionModule_RevealingPhaseOver();
```

Comment (xorsal) [reply solidity/contracts/modules/resolution/PrivateERC20ResolutionModule.sol:89]: Understandable.

I agree it's easier to read, but it also forces me to think properly.

File: solidity/test/unit/modules/resolution/PrivateERC20ResolutionModule.t.sol [hunk 2]
```diff
@@ -173,7 +177,7 @@ contract PrivateERC20ResolutionModule_Unit_CommitVote is BaseTest {
         votingToken: token,
         minVotesForQuorum: 1,
         committingTimeWindow: 40_000,
```
Comment (xorsal) [inline solidity/test/unit/modules/resolution/PrivateERC20ResolutionModule.t.sol:179]: would it make sense to use a constant for the committing window as well?

**Review by xorsal — DISMISSED** [review]

File: solidity/test/unit/modules/resolution/PrivateERC20ResolutionModule.t.sol (hunk 2 above)
Comment (0xShaito) [reply solidity/test/unit/modules/resolution/PrivateERC20ResolutionModule.t.sol]: Added

**Review by xorsal — APPROVED** [review]

**Pull request merged**